locksmith/src/operations/metadataOperations.ts

```diff
@@ -287,19 +288,26 @@ export const getLockMetadata = async ({
 
   // Now let's see if there is an event data that needs to be attached
   // find checkout URLs that include this lock
+  // We look for both cases... because I am not sure how to look for
```

Comment:
I really need to find a way to look up inside of JSON objects in a case insensitive way.
Any idea @searchableguy @clemsos ?